tests/__init__.py

```diff
@@ -7,13 +7,14 @@
 from dbus.mainloop.glib import DBusGMainLoop
 from gi.repository import GLib, Gio
 from itertools import count
-from typing import Any, Dict, Optional, NamedTuple, Callable, List
+from typing import Any, Dict, Optional, NamedTuple, Callable, List, Tuple
```

Comment:
ftr python 3.9 let us use `list`, `dict`, `tuple` as types and instead of `Optionals` we can now use `list | None` as type.

Reply:
I'll try to remember when I touch those things the next time.